cucumber/api/features/step_definitions/request_steps.rb

```diff
@@ -162,6 +162,16 @@
   expect(@content_type).to match(content_type)
 end
 
+Then(/^the HTTP response is base64 encoded$/) do
+  @result = Base64.strict_decode64(@result)
+  expect(JSON.parse(@result).is_a?(Hash)).to be true
+end
+
+Then(/^the user "([^"]*)" is successfully authenticated to Conjur$/) do |username|
+  token = ConjurToken.new(@result)
+  expect(token.username).to eq(username)
+end
```

Comment:
we already have something similar here: https://github.com/cyberark/conjur/blob/master/cucumber/_authenticators_common/features/step_definitions/authn_common_steps.rb#L1

can we use it?

Reply:
these step is not accessible from api profile. I'll look for shared steps file solution

Reply:
you can load these steps in `cucumber.yml` under the `api` profile. you can see how we do it in other profiles.

Reply:
yes already found and updated